examples/compiler/01_iet.ipynb

```diff
@@ -0,0 +1,696 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial describes one particular intermediate representation used by the Devito Compiler: the Iteration/Expression Tree (IET), a special type of Abstract Syntax Tree (AST).\n",
+    "\n",
+    "# Part I - Top Down\n",
+    "\n",
+    "First, let's describe a $Domain$, and a $Function$ that will allow us to specify how such a domain gets modified. In particular, we will look at Functions that change through $time$. \n",
+    "\n",
+    "Thus we need a Function object with which we can build a timestepping scheme. For this purpose Devito provides so-called `TimeData` objects that encapsulate Functions that are differentiable in space and time, which are derived from basic $SymPy$ Functions. \n",
+    "\n",
+    "With this we can derive symbolic expressions for the backward derivatives in space directly via the `u.dxl` and `u.dyl` shorthand expressions (the l indicates \"left\" or backward differences) and the shorthand notation `u.dt` provided by TimeData objects to derive the forward derivative in time.\n",
```

Comment:
"TimeData" is obsolete syntax. Perhaps from more than one year ago. Use \`TimeFunction\` instead. Also, check if it appears somewhere else...